docs/source/examples/simulate_RFB_example.ipynb

```diff
@@ -846,10 +846,10 @@
    "source": [
     "## Example: Symmetric cell with crossover, where $P_{ox} \\neq P_{red}$. Chemical degradation of reduced species\n",
     "\n",
-    "We define a CCCV protocol with the `ConstantCurrentConstantVoltage` class for a **symmetric cell**, to charge up to 0.2 V and discharge down to -0.2 V, at $\\pm$ 100 mA. We set the current cutoffs to be $\\pm$ 5 mA.\\\n",
-    "We now include a crossover mechanism via the `Crossover` class, with a membrane thickness of $\\mathrm{50~\\mu m}$, and permeabilities for oxidized and reduced species: $P_{ox}=1\\times10^{-7}~cm^{2}/s~,~P_{red}=5\\times10^{-7}~cm^{2}/s$.\n",
+    "We define a CCCV protocol with the [`ConstantCurrentConstantVoltage`](https://ericfell-rfbzero.readthedocs.io/en/latest/experiment.html#src.rfbzero.experiment.ConstantCurrentConstantVoltage) class for a **symmetric cell**, to charge up to 0.2 V and discharge down to -0.2 V, at $\\pm$ 100 mA. We set the current cutoffs to be $\\pm$ 5 mA.\\\n",
+    "We now include a crossover mechanism via the [`Crossover`](https://ericfell-rfbzero.readthedocs.io/en/latest/crossover.html#src.rfbzero.crossover.Crossover) class, with a membrane thickness of $\\mathrm{50~\\mu m}$, and permeabilities for oxidized and reduced species: $P_{ox}=1\\times10^{-7}~cm^{2}/s~,~P_{red}=5\\times10^{-7}~cm^{2}/s$.\n",
     "<br>\n",
-    "The amount of species, in mols, that is crossing the membrane at each timestep (right hand side of **(13)** or **(14)**, multiplied by timestep) is given by the `delta_ox` and `delta_red` data from the simulation results container. A positive `delta_..` means that species are crossing from CLS to NCLS."
+    "The amount of species, in mols, that is crossing the membrane at each timestep (right hand side of **(13)** or **(14)**, multiplied by timestep) is given by the [`delta_ox`](https://ericfell-rfbzero.readthedocs.io/en/latest/crossover.html#src.rfbzero.crossover.Crossover) and [`delta_red`](https://ericfell-rfbzero.readthedocs.io/en/latest/crossover.html#src.rfbzero.crossover.Crossover) data from the simulation results container. A positive `delta_..` means that species are crossing from CLS to NCLS."
```

Comment:
Probably shouldn't have links for `delta_ox` and `delta_red` here.

Reply:
done